cobalt/h5vcc/h5vcc.idl

```diff
@@ -41,6 +41,7 @@ interface H5vcc {
   readonly attribute H5vccStorage storage;
   readonly attribute H5vccSystem system;
   readonly attribute H5vccTraceEvent traceEvent;
+  readonly attribute H5vccNetLog netLog;
```

Comment:
I've mentioned it offline, but do consider if this needs to be a standalone toplevel API - could just put it under traceEvent to reduce required boilerplate. 

It's probably cleaner this way though, fine to leave as is.